geode-core/src/main/java/org/apache/geode/internal/lang/SystemPropertyHelper.java

```diff
@@ -25,8 +22,10 @@
  */
 public class SystemPropertyHelper {
 
-  public static final String GEODE_PREFIX = "geode.";
-  public static final String GEMFIRE_PREFIX = "gemfire.";
+  @SuppressWarnings("unused")
+  public static final String GEODE_PREFIX = SystemProperty.GEODE_PREFIX;
```

Comment:
Why not remove unused `GEODE_PREFIX`?

Reply:
I agree that we should just remove it. The path forward is to use DEFAULT_PREFIX instead of GEODE_PREFIX.

Reply:
Removed.